### PR TITLE
added a new check

### PR DESCRIPTION
**Pull request by hathin**
Added unlicensed admin check to docs. Wording and external link to be reviewed by Jaime O.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/managed-desktop/get-ready/readiness-assessment-fix.md [hunk 1]
```diff
@@ -209,6 +209,13 @@ You have a security baseline profile that targets all users, all devices, or bot
 
 Make sure that any security baseline policies you have exclude Microsoft Managed Desktop devices. For steps, see [Use security baselines to configure Windows 10 devices in Intune](/mem/intune/protect/security-baselines). During enrollment, we apply a new security baseline to all Microsoft Managed Desktop devices. The **Modern Workplace Devices -All** Azure AD group is a dynamic group that we create when you enroll in Microsoft Managed Desktop, so you'll have to come back to exclude this group after enrollment. 
 
+### Unlicensed admins
+
+Microsoft Managed Desktop requires this setting to be enabled to ensure that we can continue to provide the highest quality of support to your organization. Without this setting enabled, we have experienced a lack of permissions error when interacting with customer tenants.
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/get-ready/readiness-assessment-fix.md]:
```suggestion
Microsoft Managed Desktop requires this setting to be enabled to ensure that we can continue to provide the highest quality of support to your organization. Without this setting enabled, we have experienced a lack of permissions errors when interacting with customer tenants. 
```

File: microsoft-365/managed-desktop/get-ready/readiness-assessment-fix.md [hunk 2]
```diff
@@ -209,6 +209,13 @@ You have a security baseline profile that targets all users, all devices, or bot
 
 Make sure that any security baseline policies you have exclude Microsoft Managed Desktop devices. For steps, see [Use security baselines to configure Windows 10 devices in Intune](/mem/intune/protect/security-baselines). During enrollment, we apply a new security baseline to all Microsoft Managed Desktop devices. The **Modern Workplace Devices -All** Azure AD group is a dynamic group that we create when you enroll in Microsoft Managed Desktop, so you'll have to come back to exclude this group after enrollment. 
 
+### Unlicensed admins
+
+Microsoft Managed Desktop requires this setting to be enabled to ensure that we can continue to provide the highest quality of support to your organization. Without this setting enabled, we have experienced a lack of permissions error when interacting with customer tenants. 
+
+**Not ready**
+
+Allow access to unlicensed admins should be enabled. For steps, see [Prerequisites for guest accounts](https://docs.microsoft.com/microsoft-365/managed-desktop/get-ready/guest-accounts?view=o365-worldwide)
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/get-ready/readiness-assessment-fix.md]:
```suggestion
**Allow access to unlicensed admins** should be enabled. For the required steps, see [Prerequisites for guest accounts](/microsoft-365/managed-desktop/get-ready/guest-accounts?view=o365-worldwide).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @hathin, kindly commit the suggested changes if you agree with them. Many thanks.

File: microsoft-365/managed-desktop/get-ready/readiness-assessment-fix.md (hunk 2 above)
Comment (hathin) [reply microsoft-365/managed-desktop/get-ready/readiness-assessment-fix.md:218]: @jaimeo can you review?

**Review by hathin — COMMENTED** [review]
@jaimeo can you review?

File: microsoft-365/managed-desktop/get-ready/readiness-assessment-fix.md (hunk 1 above)
Comment (jaimeo) [reply microsoft-365/managed-desktop/get-ready/readiness-assessment-fix.md]: Edit is not grammatical.

**Pull request merged**